code/modules/power/supermatter/supermatter_delamination/common_delams.dm

```diff
@@ -67,7 +67,7 @@
 /datum/sm_delam/tesla
 
 /datum/sm_delam/tesla/can_select(obj/machinery/power/supermatter_crystal/sm)
-	return (sm.internal_energy > POWER_PENALTY_THRESHOLD)
+	return (sm.internal_energy > POWER_PENALTY_THRESHOLD) && (world.time - sm.activation_time > SUPERMATTER_CRITICAL_TIME)
```

Comment:
Maybe instead of a hard-cap, make it so the total moles is multiplied by a proportional value so its possible earlier, but easier later on

Reply:
Ditto the above response.